Sources/Operator/local_operator.hpp

```diff
@@ -201,27 +203,39 @@ class LocalOperator : public AbstractOperator {
   void FindConn(VectorConstRefType v, std::vector<Complex> &mel,
                 std::vector<std::vector<int>> &connectors,
                 std::vector<std::vector<double>> &newconfs) const override {
+    return this->FindConn(v, mel, connectors, newconfs, true);
+  }
+
+  void FindConn(VectorConstRefType v, std::vector<Complex> &mel,
+                std::vector<std::vector<int>> &connectors,
+                std::vector<std::vector<double>> &newconfs,
+                bool clear_new) const {
```

Comment:
is clear_new used somewhere?

Reply:
Good catch. It's a leftover from a previous implementation of LocalLiouvillian. It's unneeded so I'll remove it.